.pre-commit-config.yaml

```diff
@@ -88,9 +88,6 @@ repos:
           - APScheduler~=3.10.4
           - cachetools~=5.3.2
           - aiolimiter~=1.1.0
--   repo: https://github.com/aio-libs/sort-all/
-    rev: v1.2.0
-    hooks:
-    -   id: sort-all
-        name: sort-all
-        files: ^(telegram|examples|tests)/.*\.py$
+
+default_language_version:
+  python: python3.12
```

Comment:
why 3.12 instead of our minimum supported version 3.8?

Reply:
cause I'm running that version locally 😁
It shouldn't make a big difference since black and ruff have target-versions set

Reply:
my thought is rather that ruff might make us do stuff that break on older versions 🤔

Reply:
ok I'll make it 3.8 and check what changes

Reply:
oh well I need to have 3.8 installed for pre-commit to work..